_pages/production.md

```diff
@@ -0,0 +1,79 @@
+
+Please reach out to your agency IAA contact if you have any questions. If your agency does not already have an IAA, then ask your agency contact to reach out to [partners@login.gov](mailto:partners@login.gov) to begin the IAA process which can take up to 6 weeks to complete.
+
+## Staging environment
+
+Many partners choose to create a separate Staging app in our sandbox environment for testing their staging environment, because changes take effect immediately without waiting for review and deployment.
+
+If you are testing an IAL2 integration, then we also offer a Staging environment for limited testing. You must have a signed IAA in order to deploy to Staging. Our Staging environment is approved for PII which can be useful in certain test cases. However, any configuration changes in the Staging environment must be reviewed and deployed.
+
+If you wish to deploy an application to our Staging environment, then create a “Staging” configuration app like the “Production” configuration app described in the next section.
+
+## Production configuration
+
+Before you can deploy your application to the production environment, you will need to create a separate app on our dashboard that contains your production certificate, urls and logo. Here are the steps to complete your production configuration app:
+1. Create a new app on the dashboard <https://dashboard.int.identitysandbox.gov/>
+2. Enter a Friendly Name with "Production" in the title
+3. Enter the production urls and configuration into the app
+
+**Please note**: The following items are required to promote your app to production:
+
+* All production urls should have .gov .mil or a dedicated .com address and point to an ATO'd environment.
+* If your app does not have a logo, then you will need to upload one. You can find the [logo guidelines here](https://developers.login.gov/design-guidelines/#agency-logo-guidelines).
+* If this is an SAML integration (Not OpenID Connect), then please ensure that:
+  * Assertion Consumer Logout Service URL is defined.
+  * SAML Assertion Encryption is enabled.
+    * If you are using a service which does not support SAML encryption, then please send a message to [partners@login.gov](mailto:partners@login.gov) for further guidance.
+
+## Request deployment
+
+Once you have:
+1. [Received a signed IAA number](#confirm-iaa)
+2. [Created a production configuration](#production-configuration)
+
+Please submit the [login.gov production integration request form](https://share.hsforms.com/1UTxHGOu2Q0SVyz9ulknZGw3ak9e){:target='_blank'}.
+
+## Deploying changes
+
+Please update your production configuration app in the dashboard and test the changes you wish to deploy. After you have confirmed the change, then send a message to [partners@login.gov](mailto:partners@login.gov) requesting a production deployment. Please include a brief description of the change and the url to your production configuration app in the dashboard.
```

Comment:
We should follow this up with a form for requesting changes that asks for the issuer and the dashboard URL, I think, as well as a description of the change. Is it worth just doing that now?

Reply:
I added the new form you created